### PR TITLE
Madison/intersections

### PR DESCRIPTION
**Pull request by madisonmay**

### CODE REVIEW

**Comment by sihrc** [comment]
- [ ] change back to apiv2

File: lib/indico/helper.rb [hunk 1]
```diff
@@ -19,20 +19,22 @@ def self.api_handler(data, api, config)
     d['data'] = data
 
     if data.class == Array
+      if api != "apis/intersections"
```
Comment (sihrc) [inline lib/indico/helper.rb]: we can add batch anyways right?

File: lib/indico/multi.rb [hunk 2]
```diff
@@ -26,18 +28,52 @@ def self.multi(data, type, apis, allowed, batch = false, config)
         converted_apis.push(CLIENT_TO_SERVER[api])
```
Comment (sihrc) [inline lib/indico/multi.rb]: do we still want to do this?

**Comment by madisonmay** [comment]
Rebase --> merge.

**Pull request merged**